src/components/projects.ts

```diff
@@ -3,7 +3,7 @@ export const PROJECTS: {[name: string]: { name: string, title: string; keywords:
         name: 'Argo Workflows' ,
         title: 'Workflows & Pipelines',
         description: 'Container native workflow engine for Kubernetes supporting both DAG and step based workflows.',
-        link: '/projects/argo',
+        link: '/projects/argo-workflows',
```

Comment:
I don't think this fixes the issue though?

Reply:
https://github.com/argoproj/argo-site/search?q=argoproj%2Fargo

Reply:
Issues was fixed by this PR: https://github.com/argoproj/argo-site/pull/42